core/src/main/java/com/netflix/iceberg/encryption/EncryptionKeyMetadatas.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.netflix.iceberg.encryption;
+
+import java.nio.ByteBuffer;
+
+public class EncryptionKeyMetadatas {
```

Comment:
Instead of having a new class, it would be fine to add these static methods to `EncryptionKeyMetadata`. Not a big deal.

Reply:
`BaseEncryptionKeyMetadata` is in `iceberg-core` but `EncryptionKeyMetadata` is in `iceberg-api`. So factory methods creating the `BaseEncryptionKeyMetadata` need to be in `iceberg-core` as well.